cmd/dep/testdata/gom/expected_import_output.txt

```diff
@@ -0,0 +1,4 @@
+Detected Gomfile...
+Converting from Gomfile ...
+  Trying v0.8.1 (3f4c3be) as initial lock for imported dep github.com/sdboyer/deptest
```

Comment:
There should be 2 extra lines in the output, looks like it is missing the constraints and is only printing the locked projects.